libcodechecker/libclient/thrift_helper.py

```diff
@@ -105,6 +105,10 @@ def getRunData(self, run_name_filter):
         pass
 
     @ThriftClientCall
+    def getReportDetails(self, reportId):
```

Comment:
Did we miss this from the helper?

Reply:
Yes, we did.

Reply:
It wasn't **missed** per-se. I think this method was never actually used in the command-line client **ever** before.